tests/e2e/e2e_test.go

```diff
@@ -422,6 +422,41 @@ func TestE2E(t *testing.T) {
 				return scanResultIsExpected(t, f, namespace, "test-filtered-scan", compv1alpha1.ResultCompliant)
 			},
 		},
+		testExecution{
+			Name:       "TestScanWithNodeSelectorNoMatches",
+			IsParallel: true,
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+				scanName := getObjNameFromTest(t)
+				selectWorkers := map[string]string{
```

Comment:
Please rename the variable from `selectWorkers` to something like `selectNone` or similar.